test/values.ts

```diff
@@ -694,23 +693,6 @@ export const mockFiatConnectProviderInfo: FiatConnectProviderInfo[] = [
   },
 ]
 
-export const mockFiatConnectTransfers: FiatConnectTransfer[] = [
```

Comment:
moved these to TransferStatusScreen.test.tsx since it was the only file using it